services/index/indexmgr.go

```diff
@@ -766,7 +766,7 @@ func dropIndex(db database.DB, idxKey []byte, idxName string, interrupt <-chan s
 		}
 		err = dropInvalidTx(db)
 		if err != nil {
-			return err
```

Comment:
I still not confident with the swallow of the error, and why it works for our scenario. IMO, it only hides the error.